packages/server/lib/browsers/chrome.ts

```diff
@@ -403,46 +403,26 @@ const _isAUTFrame = async (frameId: string) => {
 }
 
 const _handlePausedRequests = async (client) => {
-  await client.send('Fetch.enable')
+  await client.send('Fetch.enable', {
+    // only enable request pausing for documents
+    patterns: [{
+      resourceType: 'Document',
```

Comment:
My only concern here is about the `Other` resourceType, which seems to happen sometimes on 302 redirects. However, a following document request should almost always be made, and I don't have a current reason to believe this is a problem due to origin provider tests passing with the binary + no failing cypress tests in this repo.